### ISSUE TITLE
CRAN checks

### ISSUE THREAD

**Issue by llrs** [ropensci/BaseSet#51]
oldrel windows:
```
-- FAILURE (test-element_size.R:18:5): element_size works ----------------------
  nrow(out) not equal to 2L.
  1/1 mismatches
  [1] 7 - 2 == 5
  
  -- FAILURE (test-element_size.R:19:5): element_size works ----------------------
  out$size not equal to c(0, 1).
  Lengths differ: 7 is not 2
  
  -- FAILURE (test-pull.R:29:5): pull works --------------------------------------
  `out` not equal to c("a", "b", "a2", "b2").
  'current' is not a factor
  
  -- FAILURE (test-pull.R:59:5): pull uses active --------------------------------
  `out` not equal to c("a", "b", "a2", "b2").
  'current' is not a factor
```

oldrel mac: I'm not sure what causes this, I found it also on r-hub, but couldn't reproduce on my computer. Note that on windows it gets the same error.

```
  ── FAILURE (test-pull.R:59:5): pull uses active ────────────────────────────────
  `out` not equal to c("a", "b", "a2", "b2").
  'current' is not a factor
```

**Comment by llrs** [comment]
Could reproduce on GHA:
The failing tests are:

FAILURE (test-element_size.R:18:5): element_size works
FAILURE (test-element_size.R:19:5): element_size works
FAILURE (test-pull.R:29:5): pull works
FAILURE (test-pull.R:59:5): pull uses active

**Comment by llrs** [comment]
Fixed tests on 726c295cd07682e7d5a4890c766adbe058bc1c0c
